static/styles/components.css

```diff
@@ -160,11 +160,22 @@
     width: 90px;
     margin: 0px -0.5px;
     text-align: center;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    vertical-align: bottom; /* See http://stackoverflow.com/a/43266155/ */
     padding: 3px 10px;
     background-color: #fff;
     cursor: pointer;
 }
 
```

Comment:
Right now tabs using `display: table-cell` have a very small space between them that doesn't feel quite right:

![Tabs with gap](https://cloud.githubusercontent.com/assets/7356565/24778726/7b3f9070-1b2c-11e7-8080-dbf9533557b8.png)

This is because table elements cannot have margins (and we use negative margins to remove that gap).

Thanks to @brockwhittaker, I tried using the `border-collapse` property to fix this:

```css
.informational-overlays .tab-switcher {
    border-collapse: collapse;
}
```

But it doesn't work as expected, now there is a square-cornered border around the rounded tabs, and the border between them is gone:

![Tabs with collapsed border](https://cloud.githubusercontent.com/assets/7356565/24778775/bfccf1ce-1b2c-11e7-86a6-29f6eaab5c56.png)

Any ideas on how to fix this?